src/gpu_common.c

```diff
@@ -160,14 +162,10 @@ void amd_probe(void)
 	if (adl_lib)
 		return;
 
-#if HAVE_WINDOWS_H
-	if (!(adl_lib = dlopen("atiadlxx.dll", RTLD_LAZY|RTLD_GLOBAL)) &&
+	if (!(adl_lib = dlopen("libatiadlxx.so", RTLD_LAZY|RTLD_GLOBAL)) &&
```

Comment:
Can we identify the right `.so.N` to try first?

Reply:
I tried peeking at "super" but it doesn't have that lib. Then I tried `dpkg -S libatiadlxx.so` on my Linux to no avail. So I give up.

Reply:
In fact "libatiadlxx.so.1" doesn't get any google hits!?